### PR TITLE
users can post a space

### LINKED ISSUE #2: posting a space
```
As a user
So that I can have some one stay at my place
I'd like to post my space.
```

### PR DESCRIPTION
**Pull request by reissjohnson**
closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # MakersBnBRails
 
-
+:white_check_mark:
 ```
 As a user
 So that I can find somewhere to stay
 I'd like to visit Makers-bnb.
 ```
+:construction:
```
Comment (tcpickard94) [inline README.md]: This can be removed

File: app/views/spaces/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<h1>Welcome To MakersBnB</h1>
+<% if @spaces.any? %>
+  <ul id="spaces">
+    <% @spaces.each do |space| %>
+      <li>
+      <p><%= space.name %></p>
```
Comment (tcpickard94) [inline app/views/spaces/index.html.erb]: Indentation

File: config/routes.rb [hunk 3]
```diff
@@ -12,7 +13,7 @@
   #   get 'products/:id/purchase' => 'catalog#purchase', as: :purchase
 
   # Example resource route (maps HTTP verbs to controller actions automatically):
-  #   resources :products
+     resources :spaces
```
Comment (tcpickard94) [inline config/routes.rb]: Indentation

File: app/views/spaces/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<%= form_for @space do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+  <%= f.label :location %>
```
Comment (tcpickard94) [inline app/views/spaces/new.html.erb]: Maybe a space after each of theses?

**Comment by frazerWatson** [comment]
:shipit:

**Pull request merged**